recipes/SimpleElastix/bld.bat

```diff
@@ -0,0 +1,4 @@
+
+
+
+if errorlevel 1 exit 1
```

Comment:
If you are not building on Windows just delete this file and add `skip: True  # [win]` in the build section of the `meta.yaml`.